blog/page1.md

```diff
@@ -192,19 +199,17 @@ Next, I created a static HTML page that will accept the image from the user and
 
 </html>
 ```
-**Here's what I have here:**
-* Input box for the username (simple and *insecure* auth system)
+**Above we have:**
+* Input box for the username (simple but *insecure* auth system)
 * Button to submit
 
-Now, you may have noticed that I have `<script src="js/upload.js"></script>`, that is where we're heading next...
+Now, you may have noticed that I have `<script src="js/upload.js"></script>`, that is where we're heading next... <-- Where!! Line numbers or snippets, please.
 
-### Frontend: The Javascript
+### Frontend: The Javascript <- Be more specific, what is the JS in this section for?
 
-Simply put, this block of Javascript updates the preview thumbnail while getting the picture, gets the username, and sends them both over to the function we just coded.
-1. `loadFile()` is called when the file input changes to display the thumbnail
-2. `handle()` is called when the file is submitted to POST the image and username
-    * Image is sent in the body, and username is sent as a header
+This block of Javascript updates the preview thumbnail while getting the picture, gets the username, and sends them both over to the function we just coded.
 
+First, `loadFile()` is called when the file input changes to display the thumbnail.
```

Comment:
This code sample was relatively easy to split up, so I did it for you. Use this as an example!